thirdeye/thirdeye-frontend/app/pods/manage/alert/tune/route.js

```diff
@@ -4,20 +4,50 @@
  * @exports manage/alert/edit/explore
  */
 import RSVP from "rsvp";
+import _ from 'lodash';
+import Ember from 'ember';
```

Comment:
Let's refrain from importing ember. Let's stick to the new module apis: https://github.com/emberjs/rfcs/blob/master/text/0176-javascript-module-api.md#addendum-1---table-of-module-names-and-exports-by-global 

note: I'm ok if this is done in a follow up pr

Reply:
done